lib/modules/easterEgg.js

```diff
@@ -23,11 +21,10 @@ module.go = () => {
 			notificationID: 'konami',
 		});
 		setTimeout(() => $baconBit.addClass('makeitrain'), 500);
-	};
+	});
 	konami.iphone.load = () => {}; // nix touch support; occasional false positives on touchscreen laptops
-	konami.load();
 };
 
 export function konamiActive() {
-	return konami && konami.almostThere;
+	return konami;
```

Comment:
It looks like `almostThere` was added manually by honestbleeps, since it doesn't show up anywhere [in the source](https://github.com/snaptortoise/konami-js). Alas, [it predates our Git history](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blame/3658a69c4fa1f3829ba680c45f8cc5421d266bd1~3/lib/reddit_enhancement_suite.user.js#L15685) so who knows.

We could just bring back the [old local konami.js](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/cfda25d9d0a707a63d1d490d50ba9f6feee673c5/lib/vendor/konami.js) before it was replaced with the npm module.